zkevm/Cargo.toml

```diff
@@ -47,10 +49,12 @@ itertools = "0.10.5"
 procfs = "0.13.0"
 
 [features]
-default = []
-#default = ["prove_verify"]
+# default = []
```

Comment:
Is there any reason to not just delete `default = []`? It's not hard to add back in if needed and not used. Same question applies for a lot of code I saw commented out but not deleted and was curious if there is a reason behind htis?

Reply:
"not hard to add back": adding back needs to key in more than ten letters while uncommenting it needs only press the delete key.  At least for now for us, developing experience is more important than code reading experience.